templates/store/categories/iot.html

```diff
@@ -0,0 +1,456 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1yfnQ5pXu5tDTG3M7Yv_gpjd894NUzSBKcvhB_pZqrMo/edit{% endblock %}
+
+{% block meta_title %}
+  Snapcraft - Fast and secure publishing for Internet of Things
+{% endblock %}
+
+{% block meta_description %}
+  Transactional updates, airtight security, and compatibility across architectures and OSes make snaps a perfect fit for IoT.
+{% endblock %}
+
+{% block content %}
+  <section id="main-content" class="p-strip--image snapcraft-banner-background">
+    <div class="row">
+      <div class="col-7">
+        <h2>Internet of Things</h2>
```

Comment:
This page needs a h1 tag, using `class="p-heading--two"` will allow preserving the current styling.

Reply:
Done, thanks